jax/experimental/optimizers.py

```diff
@@ -200,8 +200,8 @@ def get_params(x):
   return init, update, get_params
 
 @optimizer
-def momentum(step_size, mass):
-  """Construct optimizer triple for SGD with Nesterov momentum.
+def momentum(step_size, mass, nesterov=False):
```

Comment:
I'm a little hesitant of switching behavior based on a flag, and doubly hesitant about that flag having a default value. How about we just have two separate optimizers? Each is only a few lines, and that will make the comparison even easier (and avoid the `d` intermediate). WDYT?